src/extractPdfText.ts

```diff
@@ -1,51 +1,59 @@
 import { getPdfPagesCount } from "./getPdfPagesCount";
 import { capitalizeFirstLetter } from "./helperFunctions";
-const pdfLib = require('pdf-to-text');
-// figure out how to deal with this import vs require thing
-// tried modifying tsconfig.json target and module to es2022 and moduleResolution uncomment
-// ^ but didn't work, kept saying module couldn't be found, etc.
+import { join } from "path";
+//@ts-expect-error no type declarations for this module
+import pdfLib from "pdf-to-text";
+import { PdfLib } from "./types";
 
 // Get the user's desired pages range to translate, with a few validity checks
-const getPagesRange = async(pageFrom?: number, pageTo?: number) => {
-    const pagesCount = await getPdfPagesCount();
+const getPagesRange = async (pageFrom = 1, pageTo?: number) => {
+   const pagesCount = await getPdfPagesCount();
 
-    const inputFrom: number = pageFrom !== undefined ? pageFrom : 1;
-    const from: number = inputFrom !< 1 ? inputFrom : 1;
+   const from: number = pageFrom < 1 ? 1 : pageFrom;
 
-    const inputTo: number = pageTo !== undefined ? pageTo : pagesCount;
-    const to: number = inputTo !< pagesCount ? inputTo: pagesCount;
+   const inputTo = pageTo ?? pagesCount;
+   const to: number = inputTo < pagesCount ? inputTo : pagesCount;
 
-    return {from: from, to: to}
-}
+   return { from, to };
+};
 
 // Extract the text from the .pdf
-const getPdfText = async(pdfLib: any, pdfFilePath: string, pagesRange: {from: number, to: number}): Promise<string> => {
-    return new Promise((resolve, reject) => {
-        const pdfText: string = pdfLib.pdfToText(pdfFilePath, pagesRange, (err: string, pdfText: string) => {
-        if (err) {
-            console.log(capitalizeFirstLetter(err))
-            reject(capitalizeFirstLetter(err))
-        }
-        resolve(pdfText)
-        });
-        return pdfText;
```

Comment:
Your uppermost function already returns a Promise. This callback inside the Promise doesn't need to return anything as the return value is not used. `resolve` and `reject` are the only way to return something from this callback value. Anything `return`ed will be ignored